### DISCUSSION TITLE
Getting setup on Windows 11 - where do I find help?

### DISCUSSION THREAD

**Discussion by AJBailie** [NREL/ReEDS-2.0#123] [Q&A: Execution]
HI

I have tried to follow the README instructions but can't get ReEDs to run. It stops after:
 
Finished calculation of financial parameters for ref_seq

I am using the cases.csv with gams directory set for my machine
I tested GAMS and Python access as per Readme and seemed to work
My new run directory appears with files in the folder Input_cases
All other folders are empty (g00files, lstfiles, outputs)

I assume I have something messed up in the environment but not sure how to get help

thanks
Alison
![Screenshot 2022-06-29 152359](https://user-images.githubusercontent.com/108305064/176547620-2d622afe-4b9b-4ea9-86fe-6ae593bfe85e.jpg)

**Comment by wesleyjcole** [comment]
Alison - I recommend taking a look at the gamslog.txt file to see if that highlights an error that might not be apparent from this output.  If you can't make sense of it, feel free to send that file along to me at wesley.cole@nrel.gov and I'll take a look.
